common/lib/chem/setup.py

```diff
@@ -8,6 +8,6 @@
         "pyparsing==2.0.7",
         "numpy==1.6.2",
         "scipy==0.14.0",
-        "nltk==3.2.5",
+        "nltk",
```

Comment:
I think we need to bump the version of chem for this, otherwise existing virtualenvs might not install the new version and get the loosened requirement?

Reply:
The generated requirements file still pins it.  I want to gradually get these local apps out of the business of pinning anything.  The only potential problem I can see down the road is if we upgrade nltk later and it breaks existing virtualenvs that still remember the earlier pin here...but with devstack images now getting routinely rebuilt and pulled, that seems less likely than before (and a good reason to yank this sooner rather than later).  Am I still missing any subtleties of the issue with this?

Reply:
Probably just me being traumatized from the ways this has broken recently. It's probably fine if nobody tries to touch that version any time soon. Seems like it would be safer to force the version so we know the right thing is there, but I don't feel strongly about it.

Reply:
Why unpin nltk but not the other requirements?

Reply:
n/m, "because nltk is at the latest version"